app/controllers/discover.js

```diff
@@ -213,9 +213,19 @@ export default Ember.Controller.extend(Analytics, {
             return this.set('results', results);
         });
     },
+    totalPages: Ember.computed('numberOfResults', 'size', function() {
+        return Math.ceil(this.get('numberOfResults') / this.get('size'));
```

Comment:
I *think* this page uses SHARE search results, which are a separate network request not made to the OSF APIv2. In that case this is fine.

But if you ever need to paginate a search from the OSF API, we historically would automatically calculate a "total number of pages" value in the serializer. I mention it here in case it's useful, but also because I vaguely recall hearing someone say that the API changed, and I'm not sure the below is still relevant:
https://github.com/abought/ember-osf/blob/9bfb62aae2fc18728fea5546ab64fa4ed0aea886/addon/serializers/osf-serializer.js#L123-L123